Update peerDependencies to core@^3.0.0

**Summary**

<!--- Provide a general summary of the issue in the title above -->

The new version 3.0.0 still refers to 2.0.0 in the peerDependencies.

**Stryker config**

<!--- Please place your stryker config below. Feel free to change paths in the files and mutate arrays if you cannot share them. -->

**Test runner config**

<!--- Please Put your Jest / Karma / Jasmine / Mocha (...) configuration here. -->

**Stryker environment**

<!-- Please list your stryker plugins + versions here (output of `npm ls | grep stryker`) -->

```
├─┬ UNMET PEER DEPENDENCY @stryker-mutator/core@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
│ ├── @stryker-mutator/util@3.0.0
├── @stryker-mutator/html-reporter@3.0.0
├─┬ @stryker-mutator/javascript-mutator@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
├─┬ @stryker-mutator/mocha-framework@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
│ └── @stryker-mutator/util@3.0.0
├─┬ @stryker-mutator/mocha-runner@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
├─┬ @stryker-mutator/typescript@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
│ ├── @stryker-mutator/util@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/javascript-mutator@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/mocha-framework@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/mocha-runner@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/typescript@3.0.0
npm ERR! peer dep missing: mocha@>= 2.3.3 < 7, required by @stryker-mutator/mocha-framework@3.0.0
npm ERR! peer dep missing: mocha@>= 2.3.3 < 7, required by @stryker-mutator/mocha-runner@3.0.0
├─┬ stryker-cli@1.0.0```

<!-- Please also add the test runner you are using.  Examples:-->


**Test runner environment**

<!-- Please add your test command here (probably the command you use in `npm test`) command -->


<!-- Please add any config files that are used by your test runner. For example jest.config.js, karma.conf.js, angular.json, mocha.opts, ...  -->



**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             |
| npm              |
| Operating System |

**Add stryker.log**

<!-- Please add your stryker.log file. This file can be generated using `stryker run --fileLogLevel trace`. You can drag and drop it here. -->
<!-- Your source code is never logged to this file, however file names are logged. Feel free to obfuscate those log messages if you think it is a problem -->

--- a/packages/babel-transpiler/package.json
+++ b/packages/babel-transpiler/package.json
@@ -49,7 +49,7 @@
   },
   "peerDependencies": {
     "@babel/core": "^7",
-    "@stryker-mutator/core": "^2.0.0"
+    "@stryker-mutator/core": "^3.0.0"
   },
   "devDependencies": {
     "@babel/cli": "~7.8.3",

--- a/packages/grunt-stryker/package.json
+++ b/packages/grunt-stryker/package.json
@@ -37,7 +37,7 @@
     "@types/node": "^13.7.1"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "grunt": ">=0.4.5"
   }
 }

--- a/packages/jasmine-framework/package.json
+++ b/packages/jasmine-framework/package.json
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/stryker-mutator/stryker/tree/master/packages/jasmine#readme",
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "jasmine-core": ">=2"
   },
   "dependencies": {

--- a/packages/jasmine-runner/package.json
+++ b/packages/jasmine-runner/package.json
@@ -36,7 +36,7 @@
   },
   "homepage": "https://github.com/stryker-mutator/stryker/tree/master/packages/jasmine-runner#readme",
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "jasmine": ">=2"
   },
   "devDependencies": {

--- a/packages/javascript-mutator/package.json
+++ b/packages/javascript-mutator/package.json
@@ -43,7 +43,7 @@
     "tslib": "~1.11.1"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0"
+    "@stryker-mutator/core": "^3.0.0"
   },
   "devDependencies": {
     "@babel/types": "~7.8.0",

--- a/packages/jest-runner/package.json
+++ b/packages/jest-runner/package.json
@@ -48,7 +48,7 @@
     "react-scripts-ts": "~3.1.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "jest": ">= 22.0.0"
   },
   "dependencies": {

--- a/packages/karma-runner/package.json
+++ b/packages/karma-runner/package.json
@@ -47,7 +47,7 @@
     "karma-mocha": "^1.3.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0"
+    "@stryker-mutator/core": "^3.0.0"
   },
   "dependencies": {
     "@stryker-mutator/api": "^3.0.0",

--- a/packages/mocha-framework/package.json
+++ b/packages/mocha-framework/package.json
@@ -40,7 +40,7 @@
     "tslib": "~1.11.1"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "mocha": ">= 2.3.3 < 7"
   },
   "dependencies": {

--- a/packages/mocha-runner/package.json
+++ b/packages/mocha-runner/package.json
@@ -48,7 +48,7 @@
     "@types/node": "^13.7.1"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "mocha": ">= 2.3.3 < 7"
   }
 }

--- a/packages/typescript/package.json
+++ b/packages/typescript/package.json
@@ -50,7 +50,7 @@
     "surrial": "~1.0.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "typescript": "^3.0.3"
   },
   "initStrykerConfig": {

--- a/packages/vue-mutator/package.json
+++ b/packages/vue-mutator/package.json
@@ -43,7 +43,7 @@
     "@stryker-mutator/api": "^3.0.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "vue-template-compiler": "^2.0.0"
   }
 }

--- a/packages/wct-runner/package.json
+++ b/packages/wct-runner/package.json
@@ -35,7 +35,7 @@
     "web-component-tester": "~6.9.2"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "web-component-tester": "^6.9.0"
   },
   "author": "Nico Jansen <jansennico@gmail.com>",

--- a/packages/webpack-transpiler/package.json
+++ b/packages/webpack-transpiler/package.json
@@ -43,7 +43,7 @@
     "webpack": "~4.42.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "^2.0.0",
+    "@stryker-mutator/core": "^3.0.0",
     "webpack": ">=2.0.0"
   },
   "dependencies": {

